starmark-integration/tests/test-helpers.ts

```
@@ -35,4 +35,76 @@
+  private detectionError: Error | null = null;
+  private storeResults: import('../src/types').StorageResult[] = [];
+  private healthResult: boolean = true;
+  private analyticsResult: import('../src/types').AnalyticsData | null = null;
+
+  constructor(name: string) {
+    this.name = name;
+  }
+
+  async store(_feedback: import('../src/types').FeedbackData): Promise<import('../src/types').StorageResult> {
+    if (this.storeResults.length > 0) {
+      return this.storeResults.shift()!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)